examples/logical_interconnects.py

```diff
@@ -145,17 +146,19 @@
 pprint(internal_vlans)
 
 # Update the interconnect settings
-print("\nUpdates the interconnect settings on the logical interconnect")
-interconnect_settings = {
-    'ethernetSettings': logical_interconnect['ethernetSettings'].copy(),
-    'fcoeSettings': {}
-}
-interconnect_settings['ethernetSettings']['macRefreshInterval'] = 7
-logical_interconnect = oneview_client.logical_interconnects.update_settings(logical_interconnect['uri'],
-                                                                            interconnect_settings)
-print("Updated interconnect settings on the logical interconnect")
-print("  with attribute 'macRefreshInterval' = {macRefreshInterval}".format(**logical_interconnect['ethernetSettings']))
-pprint(logical_interconnect)
+# End-point supported only in api-versions 500 and below.
+if oneview_client.api_version <= 500:
+    print("\nUpdates the interconnect settings on the logical interconnect")
+    interconnect_settings = {
+        'ethernetSettings': logical_interconnect['ethernetSettings'].copy(),
```

Comment:
I don't think we need to use copy. 
logical_interconnect['ethernetSettings']

Reply:
This is the existing code. I have just added an `if`  as this endpoint is not supported in api-v600.